src/main/scala/com/databricks/spark/sql/perf/table.scala

```diff
@@ -47,7 +35,7 @@ abstract class TableForTest(
 
   val name = table.name
 
-  val outputDir = s"$baseDir/parquet/${name}"
+  val outputDir = s"$baseDir/${name}"
```

Comment:
The hardcoded structure with parquet is not respected by all datasets (in particular for the kms dataset that we have internally) so it was handy to be able to specify directly the path we wanted at TPCDS creation.